src/inference/src/dev/core_impl.cpp

```diff
@@ -751,7 +751,7 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::compile_model(const std::shared_ptr<
                                            parsed._config,
                                            ov::SoPtr<ov::IRemoteContext>{},
                                            cacheContent);
-        });
+        }, coreConfig.get_enable_mmap());
```

Comment:
what if `config` contains `enable_mmap(false)` while coreConfig has true for the same key?

Reply:
Lets read it always from config. It is true by default